src/test/java/seleniumEasyDemo/AlertTests.java

```diff
@@ -0,0 +1,54 @@
+package seleniumEasyDemo;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import org.testng.annotations.Test;
+
+public class AlertTests extends BaseTest {
+    protected Wait<WebDriver> wait;
```

Comment:
are you going to extend  your AlertTests class? Why do you use `protected` modifier?

Reply:
Fixed. I just forgot to change to private.